ibc-core/ics24-host/cosmos/src/upgrade_proposal/handler.rs

```diff
@@ -7,12 +7,12 @@ use tendermint::abci::Event as TmEvent;
 use super::UpgradedClientStateRef;
 use crate::upgrade_proposal::{UpgradeClientProposal, UpgradeExecutionContext, UpgradeProposal};
 
-/// Handles an upgrade client proposal
+/// Executes an upgrade client proposal.
 ///
 /// It clears both IBC client and consensus states if a previous plan was set.
 /// Then it will schedule an upgrade and finally set the upgraded client state
 /// in upgrade store.
-pub fn upgrade_client_proposal_handler<Ctx>(
+pub fn execute_upgrade_client_proposal<Ctx>(
```

Comment:
I am ok with this renaming. but I am ok with the previous naming. Can you please explain why this is refactored?

Reply:
The intention was to more closely adhere to the validation / execution structuring of ibc-rs's handlers. This function aligns with other execution handlers (as opposed to validation handlers) in terms of its scope, so I was thinking that could be better reflected in the naming.

Reply:
Makes sense! I am on-board with it.